examples/todomvc/cypress.config.ts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from "cypress";
+
+export default defineConfig({
+  e2e: {
+    baseUrl: "http://localhost:8080",
+    "defaultCommandTimeout": 30000
```

Comment:
I would prefer the default command timeout to remain `4_000` as doing so may significantly extend test runtime if many tests start to fail (which is not uncommon if we are trying to update Yew itself.)

Reply:
I changed it to 4_000